autoflake.py

```diff
@@ -569,9 +607,11 @@ def _main(argv, standard_out, standard_error):
                         help='by default, only unused standard library '
                              'imports are removed; specify a comma-separated '
                              'list of additional modules/packages')
+    parser.add_argument('--expand-single-star-import', action='store_true',
```

Comment:
Oh, never mind. I think I know what this means. It only works if there is a single start import in the whole file.